ods_ci/tests/Resources/Page/Operators/ISVs.resource

```diff
@@ -37,7 +37,8 @@ Install ISV Operator From OperatorHub Via CLI    # robocop: disable
     Wait Until Keyword Succeeds    1 min    0 sec
     ...    Is Resource Present    Subscription    ${subscription_name}     ${namespace}    ${IS_PRESENT}
     Sleep    30s
-    ${installplan_name}=     Get Resource Attribute      ${namespace}
+    ${installplan_name}=       Wait Until Keyword Succeeds    1 min    0 sec
```

Comment:
@asanzgom I think we need to refactor the keyword in the PR.
Wait Until Keyword Succeeds expects no failures in the keyword to check, but when finding an empty name for a install plan, this is not an error indeed, so the keyword will assume success in the first iteration.

Reply:
yes @CFSNM that is the final approach I took